vars/other.groovy

```diff
@@ -1,4 +1,30 @@
-def map =
+def get_current_stage(start_stage, only_stage){
```

Comment:
It's good practice to add docstrings, (or how they are called in groovy) to explain what the arguments are.
What is only_stage actually? To re-run only that stage specified by start_stage? It doesn't do that currently.

Reply:
I can add a docstrings to this function. But what about all other functions ? You want me to document them also?

Reply:
Of course no, only what this PR introduces.